WIP: Proposal of format to hold the manual curation information (#2933)
SpikeInterface/spikeinterface: doc/modules/curation.rst

    # here is the final clean sorting
    clean_sorting = cs.sorting

Manual curation format
----------------------

> SpikeInterface internally support a manual curation format JSON based.

Reviewer: ```suggestion
SpikeInterface internally supports a JSON-based manual curation format.
```

> When a mnual curation is necessary, modifying in place a dataset is a bad practice.

Reviewer: ```suggestion
When manual curation is necessary, modifying a dataset in place is a bad practice.
```

> Instead, to keep the reproducibility in the spike sorting piepline, we introduce a manual curation format,

Reviewer: ```suggestion
Instead, to ensure the reproducibility of the spike sorting pipelines, we have introduced a simple and JSON-based manual curation format.
```

> simple and JSON based. This format defines at the moment : merges + deletions + manual tags.

Reviewer: ```suggestion
This format defines at the moment : merges + deletions + manual tags.
```

The simple file can be kept along side the output of a sorter and applied on the result to have a "clean" result.

This format has two part:

  * **definition** with the folowing keys:

    * "format_version" : format specification

>     * "unit_ids" : give the list of unit_ds

Reviewer: ```suggestion
    * "unit_ids" : the list of unit_ds
```

>     * "label_definitions" : list of label category and possible labels per category.

Reviewer: ```suggestion
    * "label_definitions" : list of label categories and possible labels per category.
```

                            Every category can be *exclusive=True* onely one label or *exclusive=False* several labels possible

  * **manual output** curation with the folowing keys:

    * "manual_labels"
    * "merged_unit_groups"
    * "removed_units"

> Here the description of the format with a simple example:

Reviewer: ```suggestion
Here is the description of the format with a simple example:
```
